.pre-commit-config.yaml

```diff
@@ -0,0 +1,8 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 24.4.2
+    hooks:
+      - id: black
+        language_version: python3.12
+        args: ["--line-length=79"]
```

Comment:
This will cause issues since we test the code formatting with the default `black` settings, which means a line length of 88.

Reply:
I see, I have not tested any large lines, I changed it to default settings with no arguments.